### PR TITLE
Feature/sales

### PR DESCRIPTION
**Pull request by Peter2622**

### CODE REVIEW

File: lib/view/sales/customer_details.dart [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(8),
+        color: _appColors.whiteColor,
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            'Ajith Kumar',
+            style: TextStyle(color: _appColors.primaryColor, fontSize: 20),
```
Comment (ajithtechlambdas) [inline lib/view/sales/customer_details.dart:102]: text used from appconst

File: lib/view/sales/customer_details.dart [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+          ),
+          AppWidgetUtils.buildSizedBox(custHeight: 10),
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  _buildCustomerData('+91 9876543210', AppConstants.icCall),
```
Comment (ajithtechlambdas) [inline lib/view/sales/customer_details.dart:112]: same here

File: lib/view/sales/customer_details.dart [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+    );
+  }
+
+  Widget _buildPaymentOptions() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisAlignment: MainAxisAlignment.start,
+      children: [
+        _buildCustomRadioTile(
+          value: 'credit',
```
Comment (ajithtechlambdas) [inline lib/view/sales/customer_details.dart:244]: use string from appConst

File: lib/view/sales/sales_view_bloc.dart [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+abstract class SalesViewBloc {
+  TextEditingController get invoiceNoTextController;
```
Comment (ajithtechlambdas) [inline lib/view/sales/sales_view_bloc.dart:6]: line formation for all files

File: lib/view/sales/selected_sales_data.dart [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  final _appColors = AppColors();
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        AppWidgetUtils.buildSizedBox(custHeight: 10),
+        Text(
+          AppConstants.selectVehicleAndAccessories,
+          style: TextStyle(
```
Comment (ajithtechlambdas) [inline lib/view/sales/selected_sales_data.dart:33]: use correct font style all the file

File: lib/view/sales/selected_sales_data.dart [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+          ],
+        ),
+        AppWidgetUtils.buildSizedBox(custHeight: 10),
+        Row(
+          children: [
+            Checkbox(
+              value: widget.addSalesBloc.isDiscountChecked,
+              onChanged: (value) {
+                setState(() {
+                  widget.addSalesBloc.isDiscountChecked = value!;
```
Comment (ajithtechlambdas) [inline lib/view/sales/selected_sales_data.dart:100]: why here use the non nullable

**Review by ajithtechlambdas — CHANGES_REQUESTED** [review]

File: lib/view/sales/customer_details.dart (hunk 2 above)
Comment (Peter2622) [reply lib/view/sales/customer_details.dart:112]: its a dummy data once when i get the data from the api ill clear that

**Pull request merged**